Source/Parser/ParserHelpers.py

```diff
@@ -0,0 +1,88 @@
+# the file is very WIP, but getStringList and getSingleString already work
+
+from Parser.PyParser import peek_char, getNextLexeme, getNextTokenWithoutMatching, registerKeyword, parseStream, \
+    registerRegex
+
+
+def doNothing(unused, theStream):
+    pass
+
+
+def ignoreItem(unused, theStream):
+    nextl = getNextLexeme(theStream)  # equals
+    if nextl == '=':
+        nextl = getNextLexeme(theStream)
+    if nextl == '{':
+        braceDepth = 1
+        while True:
+            if not peek_char(theStream):  # I use this instead of eof
+                return
+            token = getNextLexeme(theStream)
+            if token == '{':
+                braceDepth += 1
+            elif token == '}':
+                braceDepth -= 1
+                if braceDepth == 0:
+                    return
+
+
+def getIntList(theStream):
+    ints = []
+
+    def intListFun1(theInt, theStream):
+        ints.append(int(theInt))
+
+    registerKeyword('\d+', intListFun1)
+
+    def intListFun2(theInt, theStream):
+        newInt = theInt[1: len(theInt) - 1]
+        ints.append(int(newInt))
+
+    registerKeyword('"\d+"', intListFun2)
+    parseStream(theStream)
+
+    return ints
+
+
+def getSingleInt(theStream):
+    getNextTokenWithoutMatching(theStream)  # gets equals sign
+    token = getNextTokenWithoutMatching(theStream)
+    if token[0] == "\"":
+        token = token[1: len(token) - 1]
+    try:
+        theInt = int(token)
+    except TypeError:
+        print("Expected an int, but instead got " + token)
+    return theInt
+
+
+# possible additions:
+# doubleList
+# singleDouble
+
+
+def getStringList(theStream):
+    strings = []
+    regex0 = r'""'
+    regex1 = r'["](.*?)["]'
+
```

Comment:
you can have string lists with quoted names: 
male_names = {Joe "Mr. Fuzzy" Jack}
Please test this works for that too.

Reply:
There was indeed a problem here, I fixed it by using this regex instead: `(".*?")|([.\w]+)`.
First capturing group is for quoted names, second one for others.
I added the list of male names here:
![image](https://user-images.githubusercontent.com/29546927/78161030-24f7b380-7445-11ea-959c-49e2b8794648.png)
Result:
![image](https://user-images.githubusercontent.com/29546927/78163536-d0eece00-7448-11ea-9b79-db15a546bbab.png)

Seems OK now.